### PR TITLE
Farcaster weighted voting

### PR DESCRIPTION
**Pull request by rbennettcw**
## Link to Issue
Closes: #10161

## Description of Changes
- Adds vote weight multiplier for ContestManager and calculated voting weight to ContestAction– used for Farcaster contests
- For non-prod environments, the installed action button is now suffixed with the env, e.g. "Upvote Content demo" on the demo environment
- Converts some things from BigNumber to BigInt

## Test Plan
- TODO

## Deployment Plan
N/A

## Other Considerations
N/A

### CODE REVIEW

**Review by masvelio — COMMENTED** [review]
1. when contest does not have replies, backend throws 500

image

2. Sorting does not seem to work, most recent should be in order => three, two, one
image

3. Upvoted replies but it is not reflected on the UI 
image


https://warpcast.com/masvelio/0xc4cee5dd

File: packages/commonwealth/client/scripts/views/pages/CommunityManagement/Contests/ManageContest/steps/SignTransactionsStep/SignTransactionsStep.tsx [hunk 1]
```diff
@@ -114,6 +114,14 @@ const SignTransactionsStep = ({
       exchangeToken,
     } as DeploySingleERC20ContestOnchainProps;
 
+    console.log({
```
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/pages/CommunityManagement/Contests/ManageContest/steps/SignTransactionsStep/SignTransactionsStep.tsx:117]: remove

File: libs/model/src/contest/GetFarcasterContestCasts.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { Query } from '@hicommonwealth/core';
 import * as schemas from '@hicommonwealth/schemas';
-import { BulkCastsSortType, NeynarAPIClient } from '@neynar/nodejs-sdk';
-import lo from 'lodash';
+import { NeynarAPIClient } from '@neynar/nodejs-sdk';
```
Comment (Rotorsoft) [inline libs/model/src/contest/GetFarcasterContestCasts.ts:3]: convention is to name these files as .query.ts

**Review by Rotorsoft — APPROVED** [review]

**Review by masvelio — APPROVED** [review]
tested on demo, works well!

**Pull request merged**